search-service/src/test/kotlin/com/egm/stellio/search/authorization/permission/service/PermissionServiceTests.kt

```diff
@@ -128,7 +137,10 @@ class PermissionServiceTests : WithTimescaleContainer, WithKafkaContainer() {
         permissionService.create(minimalPermission).shouldSucceed()
 
         val matchingPermissions = permissionService.getPermissions(
-            PermissionFilters(ids = setOf("urn:ngsi-ld:BeeHive:A456".toUri()))
+            PermissionFilters(
+                ids = setOf("urn:ngsi-ld:BeeHive:A456".toUri()),
+                onlyGetPermission = OnlyGetPermission.ASSIGNED
```

Comment:
surprised that you had to add `ASSIGNED` to existing tests since this behavior did not exist before.